exercises/two-fer/two_fer_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'minitest/autorun'
+require_relative 'two_fer'
+
+# Common test data version: 1.1.0 c080bdf
+class TwoFerTest < Minitest::Test
+  def test_no_name_given
+    # skip
+    assert_equal "One for you, one for me.", TwoFer.two_fer
```

Comment:
When I run the generator for this problem it gives me this line: 

`assert_equal "One for you, one for me.", TwoFer.two_fer(nil)`

The generator should generate the test file without it requiring any further editing.